src/ape_run/_cli.py

```diff
@@ -60,16 +60,15 @@ def _module_str(filepath: Path) -> str:
 
             with use_temp_sys_path(filepath.parent.parent):
                 try:
-                    path = _module_str(filepath)
-                    ns = run_module(f"scripts.{path}")
+                    cli_ns = run_module(_module_str(filepath))
```

Comment:
pycharm was made about ns being overriden in places from the outerscope.
seemed easiest to use new var names